api/app/schemas/morecast_v2.py

```diff
@@ -126,6 +126,13 @@ class WeatherIndeterminate(BaseModel):
     precipitation: Optional[float] = None
     wind_direction: Optional[float] = None
     wind_speed: Optional[float] = None
+    fine_fuel_moisture_code: Optional[float] = None
+    duff_moisture_code: Optional[float] = None
+    drought_code: Optional[float] = None
+    initial_spread_index: Optional[float] = None
+    build_up_index: Optional[float] = None
+    fire_weather_index: Optional[float] = None
+    danger_region: Optional[int] = None
```

Comment:
Do we use this index anywhere (it's new to me)? I've only seen us use danger rating before.

Reply:
Ah yes, meant to bring this up this morning to make sure I was grabbing the correct entry from the api. I very well may not be, it appears elsewhere in the code base we use `dangerForest` from the api? There isn't an entry specifically called 'danger rating' in the WF1 api

Reply:
Ah right, it is `dangerForest`, I asked this question awhile ago too haha: https://chat.developer.gov.bc.ca/channel/wildfire-wfwx?msg=biLJvdkmdvHYuXk8m

Reply:
Ah awesome, thank you!